src/ngraph/check.hpp

```diff
@@ -66,14 +66,17 @@ namespace ngraph
 
 // TODO(amprocte): refactor so we don't have to introduce a locally-scoped variable and risk
 // shadowing here.
-#define NGRAPH_CHECK(exc_class, ctx, check, ...)                                                   \
+#define NGRAPH_CHECK_HELPER(exc_class, ctx, check, ...)                                            \
     do                                                                                             \
     {                                                                                              \
         if (!(check))                                                                              \
         {                                                                                          \
             ::std::stringstream ss___;                                                             \
-            ::ngraph::write_all_to_stream(ss___, __VA_ARGS__);                                     \
+            ::ngraph::write_all_to_stream(ss___, ##__VA_ARGS__);                                   \
```

Comment:
This `##` here is the "token-pasting hack" I was referring to in the PR comment. It basically means "backspace the comma if the number of varargs is zero."

Reply:
@aslepko can you run Windows CI on this PR?